sklearn/cluster/optics_.py

```diff
@@ -86,19 +86,33 @@ class OPTICS(BaseEstimator, ClusterMixin):
     metric_params : dict, optional (default=None)
         Additional keyword arguments for the metric function.
 
-    cluster_method : string, optional (default='dbscan')
+    cluster_method : string, optional (default='xi')
         The extraction method used to extract clusters using the calculated
-        reachability and ordering. Possible values are "dbscan".
+        reachability and ordering. Possible values are "xi" and "dbscan".
 
-    eps : float, optional (default=0.5)
+    eps : float, optional (default=None)
         The maximum distance between two samples for them to be considered
-        as in the same neighborhood. Used ony when `cluster_method='dbscan'`.
-
-    min_cluster_size : int > 1 or float between 0 and 1 (default=0.005)
+        as in the same neighborhood. By default it assumes the same value as
+        ``max_eps``.
+        Used only when ``cluster_method='dbscan'``.
+
+    xi : float, between 0 and 1, optional (default=0.05)
+        Determines the minimum steepness on the reachability plot that
+        constitutes a cluster boundary. For example, an upwards point in the
+        reachability plot is defined by the ratio from one point to its
+        successor being at most 1-xi.
+        Used only when ``cluster_method='xi'``.
+
+    predecessor_correction : bool, optional (default=True)
```

Comment:
Is it worth making this parameter, rather than leaving it True all the time?

Reply:
I wouldn't oppose removing the parameter (at least for now) and putting it back if we see demand.

Reply:
No strong opinion, slightly prefer to keep it since it's not included in the original paper (R dbscan/JAVA ELKI also have this parameter)